recipes/pylogit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pylogit" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "a6809e2adb3063fe1408696080217440ab6ea5eb1f3aab62eba9e215559e2fbc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Can this be `noarch: python`?

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
It's not totally clear to me from that doc. This package uses the `future` package for python 2+3 compatibility. I'm not sure what role its dependencies such as pandas etc play in this decision.

Reply:
I added noarch: python